test/test_utils.py

```diff
@@ -179,3 +180,38 @@ def test_seconds_since_none(self, datetime_mock):
         datetime_mock.now.return_value = datetime(2016, 9, 1, 2, 0)
 
         self.assertRaises(TypeError, seconds_since, None)
+
+
+class TestFromOptionalDependency(unittest.TestCase):
+    def test_existing_module_works(self):
+        a, = FromOptionalDependency('time').do_import('sleep')
```

Comment:
I've thought about it, but I opted for consistently returning a `tuple` in order to not create special cases. Maybe it's better to allow the special case of 1.